nextjs-sidecar/host/src/pages/index.js

```diff
@@ -0,0 +1,35 @@
+import { lazy, Suspense } from "react";
+
+const RemoteComponent = ({ scope, module, ...props }) => {
+  if (!global[scope]) {
+    return null;
+  }
+
+  global[scope].init({
+    react: {
+      get: () => Promise.resolve().then(() => () => require("react")),
+      loaded: true,
+      singleton: true,
+      version: [16, 13, 1],
```

Comment:
I will need to validate this works with webpack beta21, we have changed the sharing structure. Should be good for now, but test this against your own repo (or mine with the latest webpack release)